2019-08-08 13:36:04,946 INFO: Microblog startup [in c:\users\lenovo_owner\microblog\venv\app\__init__.py:46]
2019-08-08 13:36:19,137 ERROR: Exception on /edit_profile [POST] [in c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "c:\users\lenovo_owner\microblog\venv\app\routes.py", line 103, in edit_profile
    return redirect(url_for('profile'))
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\flask\helpers.py", line 358, in url_for
    endpoint, values, method=method, force_external=external
  File "c:\users\lenovo_owner\microblog\venv\lib\site-packages\werkzeug\routing.py", line 2020, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'profile'. Did you mean 'edit_profile' instead?
2019-08-08 13:37:59,081 INFO: Microblog startup [in c:\users\lenovo_owner\microblog\venv\app\__init__.py:46]
